Swen/Classes/Swen.swift

```diff
@@ -86,18 +118,25 @@ fileprivate extension Swen {
         listeners.append(listener)
         if let sticky = sticky {
             listener.queue.addOperation {
-                listener.post(sticky)
+                handler(sticky)
```

Comment:
So we can just call listener.post ?
If so you can change that and i'll approve :)